packages/unmock-node/src/__tests__/backend/index.test.ts

```diff
@@ -8,18 +8,27 @@ const servicesDirectory = path.join(__dirname, "..", "loaders", "resources");
 describe("Node.js interceptor", () => {
   describe("with petstore in place", () => {
     let nodeInterceptor: NodeBackend;
-    beforeEach(() => {
+    beforeAll(() => {
       nodeInterceptor = new NodeBackend({ servicesDirectory });
       const unmockOptions = new UnmockOptions();
       nodeInterceptor.initialize(unmockOptions);
     });
-    afterEach(() => {
+    afterAll(() => {
```

Comment:
Cannot use `beforeEach` and `afterEach` here if tests run in parallel and share the same interceptor as the backend is stateful now. Not sure if this could break user tests as well in some cases.